gulp/tasks/browserSync.js

```diff
@@ -1,7 +1,11 @@
-var gulp            = require('gulp'),
-    browserSync     = require('browser-sync'),
-    config          = require('../config').browserSync;
+var gulp = require('gulp'),
+    browserSync = require('browser-sync'),
+    config = require('../config').browserSync;
 
-gulp.task('browserSync', ['jekyll-build'], function() {
-    browserSync(config);
-});
+(function() {
```

Comment:
I see now that you're adding them. Is that to make ESLint happy? Do we need to define a rule override for the gulp directory?